Extension/tsconfig.json

```diff
@@ -2,6 +2,7 @@
 	"compilerOptions": {
 		"module": "commonjs",
 		"target": "es6",
+		"declaration": false,
```

Comment:
The default value of `declaration` is false. Was there confusion that this needed to be included?

Reply:
Nope, I was experimenting and checked this in by accident :)